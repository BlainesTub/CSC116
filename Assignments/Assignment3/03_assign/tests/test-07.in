7
0 1 8.0
1 3 3.0
1 5 1.0
4 6 1
5 6 5.0
2 6 8
