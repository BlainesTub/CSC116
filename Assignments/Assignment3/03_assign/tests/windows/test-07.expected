Input matrix. Size : 7 x 7
 -  :   8 :  -  :  -  :  -  :  -  :  -  : 
  8 :  -  :  -  :   3 :  -  :   1 :  -  : 
 -  :  -  :  -  :  -  :  -  :  -  :   8 : 
 -  :   3 :  -  :  -  :  -  :  -  :  -  : 
 -  :  -  :  -  :  -  :  -  :  -  :   1 : 
 -  :   1 :  -  :  -  :  -  :  -  :   5 : 
 -  :  -  :   8 :  -  :   1 :   5 :  -  : 

Input Adjacency list. Size : 7
From City: A -> (B,8) 
From City: B -> (A,8) (D,3) (F,1) 
From City: C -> (G,8) 
From City: D -> (B,3) 
From City: E -> (G,1) 
From City: F -> (B,1) (G,5) 
From City: G -> (C,8) (E,1) (F,5) 

Minimum Spanning Tree. Size : 7
From City: A -> (B,8) 
From City: B -> (A,8) (D,3) (F,1) 
From City: C -> (G,8) 
From City: D -> (B,3) 
From City: E -> (G,1) 
From City: F -> (B,1) (G,5) 
From City: G -> (C,8) (E,1) (F,5) 

Cost of original matrix: 26
Cost of minimum spanning tree: 26
